src/hpke/info.rs

```diff
@@ -7,16 +7,18 @@ const DOMAIN: &str = "private-attribution";
 /// Represents the [`info`] part of the receiver context, that is: application specific data
 /// for each encryption.
 ///
-/// IPA uses key identifier, event epoch and both helper and site origins to authenticate match key
-/// encryption. It is not guaranteed that the same receiver can be used for anything else.
+/// IPA uses key identifier, key event epoch, helper and match key provider origins, and
+/// site registrable domain to authenticate the encryption of a match key.
+/// It is not guaranteed that the same receiver can be used for anything else.
 ///
 /// [`info`]: https://www.rfc-editor.org/rfc/rfc9180.html#name-creating-the-encryption-con
 #[derive(Clone)]
 pub struct Info<'a> {
     pub(super) key_id: KeyIdentifier,
     pub(super) epoch: Epoch,
+    pub(super) match_key_provider_origin: &'a str,
     pub(super) helper_origin: &'a str,
-    pub(super) site_origin: &'a str,
+    pub(super) site_registrable_domain: &'a str,
```

Comment:
```suggestion
    pub(super) site_domain: &'a str,
```

This might be easier to type for us.